README.md

```diff
@@ -74,6 +74,21 @@ Apart from that, XSStrike has crawling, fuzzing, parameter discovery, WAF detect
 - [Can I copy it's code?](https://github.com/s0md3v/XSStrike/wiki/FAQ#can-i-copy-its-code)
 - [What if I want to embed it into a proprietary software?](https://github.com/s0md3v/XSStrike/wiki/FAQ#what-if-i-want-to-embed-it-into-a-proprietary-software)
 
+### Docker Build
+
+ ```
+ $ docker build -t <IMAGE NAME> .
+ OR
+ $ docker build -t xshuden/xsstrike .
```

Comment:
```suggestion
 $ docker build -t s0md3v/xsstrike .
```
to indicate from which repo xsstrike comes from

same change applies to the Docker image references below